libs/eo/origin-of-energy/shell/src/lib/eo-origin-of-energy-shell.component.ts

```diff
@@ -14,45 +14,183 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { CommonModule } from '@angular/common';
 import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
-import { EoInlineMessageScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
-import { WattIconModule, WattIconSize } from '@energinet-datahub/watt';
-
+import { MatCardModule } from '@angular/material/card';
+import { EoMediaModule } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { EoPieChartScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
-  selector: 'eo-metering-points-shell',
+  selector: 'eo-origin-of-energy-shell',
   styles: [
     `
       :host {
         display: block;
+        max-width: 1040px; /* Magic UX number */
+      }
+
+      .chart-row {
+        display: flex;
+        margin-bottom: var(--watt-space-l);
+      }
+
+      .chart-card {
+        max-width: 584px; /* Magic UX number */
+        width: 100%;
+      }
+
+      .chart-box {
+        margin-top: var(--watt-space-m);
+        margin-right: calc(var(--watt-space-xl) - var(--watt-space-s));
+        margin-left: calc(var(--watt-space-xl) - var(--watt-space-s));
```

Comment:
issue: Unless the design dimensions are too *magic* and the UX designer insists on magic spacing dimensions, we should use the Watt spacing primitives for spacing, that is Inset, Squished Inset, Stretched Inset, Stack, and Inline.

Reply:
I have remove this, and replaced it with watt-space-inset-squish-l after agreeing with UX about it :)